runtime/oti/j9nonbuilder.h

```diff
@@ -1966,6 +1966,7 @@ typedef struct J9BCTranslationData {
 #define BCT_AnyPreviewVersion  0x100000
 #define BCT_EnablePreview 0x200000
 #define BCT_DumpMaps  0x40000000
+#define BCT_Unsafe  0x80000000
```

Comment:
This overlaps the mask:
```
#define BCT_MajorClassFileVersionMask  0xFF000000
```
Need to look for free bits earlier in the field

Reply:
There is no unused bits  out there
```
#define BCT_LittleEndianOutput  0
#define BCT_BigEndianOutput  1
#define BCT_StripDebugAttributes  0x100
#define BCT_StaticVerification  0x200
#define BCT_DumpPreverifyData  0x400
#define BCT_Xfuture  0x800
#define BCT_AlwaysSplitBytecodes 0x1000
#define BCT_IntermediateDataIsClassfile  0x2000
#define BCT_ValueTypesEnabled  0x4000
#define BCT_StripDebugLines  0x8000
#define BCT_StripDebugSource  0x10000
#define BCT_StripDebugVars  0x20000
#define BCT_StripSourceDebugExtension  0x40000
#define BCT_RetainRuntimeInvisibleAttributes  0x80000
#define BCT_AnyPreviewVersion  0x100000
#define BCT_EnablePreview 0x200000
#define BCT_DumpMaps  0x40000000
```
Probably setting to `0x100000000` BCT_Unsafe is more desirable to avoid the overlapping with the class version bits.
```
#define BCT_MajorClassFileVersionMask  0xFF000000
#define BCT_MajorClassFileVersionMaskShift  24
#define BCT_Java2MajorVersionShifted  0x2E000000
#define BCT_Java5MajorVersionShifted  0x31000000
#define BCT_Java6MajorVersionShifted  0x32000000
#define BCT_Java7MajorVersionShifted  0x33000000
#define BCT_Java8MajorVersionShifted  0x34000000
#define BCT_Java9MajorVersionShifted  0x35000000
#define BCT_Java10MajorVersionShifted 0x36000000
#define BCT_Java11MajorVersionShifted 0x37000000
#define BCT_Java12MajorVersionShifted 0x38000000
#define BCT_Java13MajorVersionShifted 0x39000000
#define BCT_Java14MajorVersionShifted 0x3A000000
#define BCT_Java15MajorVersionShifted 0x3B000000
#define BCT_JavaMaxMajorVersionShifted BCT_Java15MajorVersionShifted
```

Reply:
This won't work as `flags` is `U_32`
```
j9bcutil_readClassFileBytes(J9PortLibrary *portLib,
	IDATA (*verifyFunction) (J9PortLibrary *aPortLib, J9CfrClassFile* classfile, U_8* segment, U_8* segmentLength, U_8* freePointer, U_32 vmVersionShifted, U_32 flags, I_32 *hasRET),
	U_8* data, UDATA dataLength, U_8* segment, UDATA segmentLength, 
U_32 flags,  <------
U_8** segmentFreePointer, void *verboseContext, UDATA findClassFlags, UDATA romClassSortingThreshold)
{
```
Might need to update to UDATA if there is no better option for that but it might change everywhere for `flags`.